notifications/tasks.py

```diff
@@ -0,0 +1,42 @@
+from django.apps import apps
+
+from jobs.celery import app as celery_app
+
+from .events import EVENTS
+from .providers import PROVIDERS
+from .settings import ASYNC_QUEUE_NAME, PROVIDERS_SETTINGS
+
+
+@celery_app.task(queue=ASYNC_QUEUE_NAME)
+def dispatch_event(event_name: str, model_label: str, instance_pk: int):
+    """
+    Called from event_decorator. Get provider codes from event name, iterate each one and if provider
+    is enabled, dispatch a new async task for processing.
+    """
+    provider_codes = EVENTS[event_name]
+    for provider_code in provider_codes:
+        if not PROVIDERS_SETTINGS[provider_code]["enabled"]:
+            continue
```

Comment:
```python
if PROVIDER_SETTINGS....:
    dispatch....
```
This way you don't need the `continue`

Reply:
My vice. Often there are complex code after if statement and I prefer execute the continue command. I agree in this case is not necessary. Thanks!

Done!